pkg/service/backup_backend_local.go

```diff
@@ -164,10 +170,11 @@ func toBackupDetails(e fs.DirEntry, prefix string) model.BackupDetails {
 	if err == nil {
 		lastModified = util.Ptr(dirInfo.ModTime())
 	}
+	path := filepath.Join(prefix, e.Name())
 	return model.BackupDetails{
-		Key:          util.Ptr(filepath.Join(prefix, e.Name())),
+		Key:          util.Ptr(path),
 		LastModified: lastModified,
-		Size:         folderSize(prefix),
```

Comment:
That was the error with file size: we calculated correctly, but for the wrong folder